src/ReactiveUI.Tests/ActivationTests.cs

```diff
@@ -248,8 +247,7 @@ public void CanUnloadAndLoadViewAgain()
             locator.InitializeReactiveUI();
             locator.Register(() => new ActivatingViewFetcher(), typeof(IActivationForViewFetcher));
 
-            using (locator.WithResolver())
-            {
+            using (locator.WithResolver()) {
```

Comment:
Again, I think this matches Rebracer (`NewLines_Braces_ControlFlow` is 0) and would match ReactiveList.cs, line 391.